<launch>
   <include file="$(find pr2_arm_navigation_config)/launch/pr2_planning_environment.launch" />
   <node pkg="arm_kinematics_constraint_aware" type="arm_kinematics_constraint_aware" name="pr2_right_arm_kinematics" output="screen">    

   <!-- Collision environment parameters -->
   <remap from="robot_description" to="/robot_description" />
   <remap from="collision_map" to="collision_map_occ" />
   <remap from="collision_map_update" to="collision_map_occ_update" />
    
   <param name="pointcloud_padd" type="double" value="0.00" />

   <param name="group" type="string" value="right_arm" />
   <param name="allow_valid_collisions" type="bool" value="false" />

   <param name="use_collision_map" type="bool" value="false" />
   <param name="collision_map_safety_timeout" type="double" value="0.0" />
   <param name="joint_states_safety_timeout" type="double" value="1.0" />
   <param name="tf_safety_timeout" type="double" value="1.0" />
   <param name="bounding_planes" type="string" value="0 0 1 -0.01" />
 
   <rosparam command="load" file="$(find pr2_arm_navigation_config)/config/robot_padding.yaml" />		
   <rosparam command="load" file="$(find arm_kinematics_constraint_aware)/config/pr2_arm_kinematics.yaml" />		
  </node>
</launch>
